### DISCUSSION TITLE
The ability to download "Show all data"

### DISCUSSION THREAD

**Discussion by BTJason** [tprouvot/Salesforce-Inspector-reloaded#596] [Ideas]
It would be great when on a record and utilizing "show all data" feature if we could download that to a CSV. There are times when I need to bring data into a sandbox or vice versa, and there are quite a lot of fields, and it is cumbersome to use the data export feature.

**Comment by sawyerfm** [comment]
@BTJason Just stumbled onto this but you can just click Export from the Show All Data page and add fields(ALL) to the query that is already provided for the given record.

**Comment by BTJason** [comment]
Im not seeing a way to add all fields in one click based on what you describe.


________________________________

**Reply by tprouvot** [reply]
From data export, you can use FIELDS(ALL) instead of fields names to select all field in the query. That's a standard function in SOQL

**Comment by BTJason** [comment]
Ahhh I wasnt of that, thanks for the education
